libvcx/src/aries/handlers/issuance/issuer/issuer.rs

```diff
@@ -18,6 +18,16 @@ pub struct IssuerConfig {
     pub tails_file: Option<String>
 }
 
+#[derive(Debug, PartialEq)]
+pub enum IssuerState {
```

Comment:
I think this should be defined in `state_machine.rs`

Reply:
`state_machine` is a private module for all SMs.